packages/core/src/schema/models/evaluationResults.ts

```diff
@@ -17,7 +17,7 @@ export const evaluationResults = latitudeSchema.table(
   'evaluation_results',
   {
     id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
-    uuid: uuid('uuid').unique(),
+    uuid: uuid('uuid').unique().notNull(),
```

Comment:
This will be safe once this is fully deployed and migrated
https://github.com/latitude-dev/latitude-llm/pull/371

This should be 0 after data migration
![image](https://github.com/user-attachments/assets/7a361768-520d-4bc2-b1ed-def105903a21)